app/javascript/gobierto_data/webapp/components/sets/VisualizationsItem.vue

```diff
@@ -46,6 +49,7 @@
   </div>
 </template>
 <script>
+import Spinner from "./../commons/Spinner.vue";
```

Comment:
Question: is this Spinner different from the `Loading` component? (lib/vue-components)

Reply:
No. I change it.